h2o-algos/src/main/java/hex/glm/GLM.java

```diff
@@ -111,6 +111,7 @@ public void computeCrossValidation() {
    */
   @Override
   public void cv_computeAndSetOptimalParameters(ModelBuilder[] cvModelBuilders) {
+    if(_parms._max_runtime_secs != 0) _parms._max_runtime_secs = 0;
```

Comment:
I thought setting max_runtime_secs to 0 will disable run time restriction.  Why are you setting it here?

Reply:
@wendycwong , this if for consistency with other algos which do the same timeout cancellation in this method: this cancellation applies only for the main model, which means that currently (and for some reason not fully clear to me) when we compute a model with CV=5 for example, with a time limit of (to make it simple) 12min, we assign 2 min (effective) for each CV model, and 2 min for the main model, but those last 2 min are kind of virtual as we will effectively use as much time as necessary...

Please see other algos for comparison: if in GLM's case, there is a better reason to keep the time limit for the main model, then I'll be happy to revert this change and add a comment explaining why this is not necessary here.

Reply:
@michalkurka do you know why we disable timeout for main model when training with CV for most algos? 
Is it really necessary? also, did we forget to disable it on GLM or was it missing on purpose?

Reply:
@seb-h2o Are you referring to GBM/DRF and Deep Learning which are doing this?

    if( _parms._stopping_rounds == 0 && _parms._max_runtime_secs == 0) return; // No exciting changes to stopping conditions

`cv_computeAndSetOptimalParameters` in those models takes into consideration the CV models and builds the main model bases on the parameters of the CV models - it calculates the average number of trees or epochs in deep learning.

The main model should not be restricted by time because you want to build a good model, not eg. an empty model while your CV models are good.

GLM is currently not using `max_runtime_secs` AFAIK

Reply:
sorry, GLM is using max_runtime_secs

Reply:
@michalkurka Yes, just wanted to have the confirmation that we should therefore also disable `max_runtime_secs` for CV main model in GLM as well

Reply:
yes, disabling the limit here makes sense to me